docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -4,6 +4,11 @@ description: Moderated Sessions
 h1: Moderated Sessions
 ---
 
+<Details title="Version Requirements" min="8.3" opened>
+  Moderated Sessions is only available in Teleport 8.3.0 and above.
+  If you are using a previous version of Teleport, you must specify `version: v4` or earlier for role resources.
```

Comment:
This sentence makes it sound like if you use an older version of Teleport and specify v4 that you can still use moderated sessions.

How about something like:

```
Moderated Sessions require `v5` roles, which are only available in Teleport 8.3.0 and above.
```